registry.json

```diff
@@ -3116,5 +3116,19 @@
  "telegram": "pgwada",                                             
  "operatorName": "Wadi",                                   
  "github": "pgwadapool"                                                     
+},
+  {
+ "ticker": "AAAU1",                                                       
+ "poolId": "64f6bd547b5ec3c35e18d1b09956af8dd2e53b7da7a9b7f58f3c5580",  
+ "social": {                                                            
+   "twitter": "Richard79337561",
+   "facebook": "@adabankau",  
+   "instagram": " ",
+   "website": "https://adabank.com.au",
+   "youtube": " "
+ },
+ "telegram": "ADABankAU",                                             
+ "operatorName": "OffTheRicta",                                   
```

Comment:
Please change this to either your real full name, real first name or leave blank